Simplenote/Classes/SPAuthViewController.swift

```diff
@@ -443,6 +445,16 @@ private extension SPAuthViewController {
         present(alertController, animated: true, completion: nil)
     }
 
+    func presentUserUnverifiedError(error: SPAuthError) {
+        let alertController = UIAlertController(title: error.title, message: error.message, preferredStyle: .alert)
+        alertController.addCancelActionWithTitle(AuthenticationStrings.unverifiedCancelText)
+        alertController.addDefaultActionWithTitle(AuthenticationStrings.unverifiedActionText) { _ in
+            AccountRemote().verify(email: self.email) { _ in }
```

Comment:
I think we should show an activity indicator (can use the one we have for account deletion) and handle errors similar to what we're doing in `AccountVerificationViewController`